spec/requests/api/v1/users/login_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+require "rspec/json_expectations"
```

Comment:
Maybe we should move this to the `spec_helper` so it's available for every test and we don't have to include it every time